### PR TITLE
Feature/02 add eslint

### PR DESCRIPTION
**Pull request by sshkatulov**

### CODE REVIEW

File: my-app/src/App.tsx [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```
Comment (i-nemets) [inline my-app/src/App.tsx]: why do you removed React ?

Comment (sshkatulov) [reply my-app/src/App.tsx]: the import is not used

Comment (i-nemets) [reply my-app/src/App.tsx]: https://ru.reactjs.org/docs/jsx-in-depth.html

Comment (sshkatulov) [reply my-app/src/App.tsx]: added the import

**Review by i-nemets — APPROVED** [review]

**Pull request merged**